### ISSUE TITLE
Implement click dummy to implement button functionality

### ISSUE THREAD

**Issue by i315315** [amosproj/amos2022ws02-automotive-test-app#38]
## User story
1.	As a Developer
2.	I want to be able to start implementing the functionality of a button even though the final steering wheel GUI is not finished yet
3.	So that I can work on implementing the functionality of a button at the same time as another developer is developing the actual steering wheel GUI. 

With this User story, a click dummy to implement button functionality is to be developed.
It can be a simple list of buttons that will later be moved to the actual steering wheel-like GUI.
The click dummy should have speaking IDs for buttons that can be reused in the actual implementation of the buttons later on.

The following buttons are needed (the list might be extended with upcoming requirements):
* SkipForward
* SeekForward (natively available from Android)
* Play/Pause
* Voicecontrol
* Phone (Event „HeadsetHook()")

## Acceptance criteria
* A click dummy to implement button functionalities is developed.
* The click dummy features all needed buttons (see list above).

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by i315315** [comment]
@Andy3189 Das Issue steht bereits auf closed, obwohl Issues erst im Review auf closed und dann ins Feature Archive geschoben werden. Außerdem steht es noch in der Swimlane "in Progress". Das liegt aber vermutlich an dem Automatismus, den ich Euch am Mittwoch gezeigt habe, oder? 
Bitte das Issue nochmal öffnen und in die richtige Swimlane verschieben.

**Comment by Andy3189** [comment]
kann sein dass das durch den pr direkt geschlossen wurde

**Comment by i315315** [comment]
> kann sein dass das durch den pr direkt geschlossen wurde

Ah ok. Ja, das kann sein. Danke fürs Reopening.
